content/en/apps/guides/database/muting_in_dashboards.md

```diff
@@ -0,0 +1,78 @@
+When a contact gets muted, two of [many things](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#muting) happen:
+
+- The target contact and all of its descendants have a `muted` property set equal to the date they were muted
+- an entry is added to the contact's `muting_history` in sentinel's info docs
+
+When building dashboards on KF, you might need to exclude these muted contacts from analytics. An easy way to do this is to check the contact's `muted` property which when present has the date value of when the contact was muted and when absent means that the contact is not muted. This works work when you're only interested in the seeing the latest data but it gets a little bit complicated when you want to look at a contact's mute state from a certain period in the past. 
+
+For example, Say a contact was muted on February and unmuted on May; If you check the contact's mute state in March from June, you'd find that the contact would not have the muted property as it would have been removed during the `unmute` in May. This is where the `muting_history` comes in. A contact's `muting_history` contains all mute and unmute events stored in a json array. An example of the mute/unmute entries
+
+```json
+{
+    "_id": "_id",
+    "_rev": "3-01ecfdd2958baeaf16fc621c5622f4a9",
+    "type": "info",
+    "doc_id": "doc-id",
+    "transitions": {
+    },
+    "muting_history": [
+        {
+            "date": "2021-04-07T13:41:09.769Z",
+            "muted": true,
+            "report_id": "_id"
+        },
+        {
+            "date": "2021-06-07T13:41:09.769Z",
+            "muted": false,
+            "report_id": "_id"
+        }
+    ],
+    "latest_replication_date": "2020-10-30T15:46:45.482Z",
+    "initial_replication_date": "2020-10-30T15:46:45.482Z"
+}
+```
+
+If we extract this data into a separate table we can get a timeline we can use to check if the contact was muted at a certain point in the past. An example of this approach can be found [here](https://github.com/medic/config-moh-kenya-app/blob/moh-kenya-safari-doctors/postgres/matviews/contactview_muted.sql) where we first extract the data using
```

Comment:
Use pronouns consistently to maintain the tone throughout the guide. I'd also remove the link to specific project code since this may change or be removed and would make it harder to maintain the links.
```suggestion
If you extract this data into a separate table you can get a timeline that you can use to check if the contact was muted at a certain point in the past. An example of this approach is shown below where the data is first extracted using the query:
```